internal/cmd/beta/server/start/start.go

```diff
@@ -0,0 +1,119 @@
+package start
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	iaasUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/spinner"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas/wait"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	serverIdArg = "SERVER_ID"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ServerId string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "start",
+		Short: "Starts an existing server or allocates the server if deallocated",
+		Long:  "Starts an existing server or allocates the server if deallocated.",
+		Args:  args.SingleArg(serverIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(
+				`Starts an existing server with ID "xxx"`,
```

Comment:
Nitpick: for the examples, they are always in the form of "Start" (instead of "Starts")

Reply:
Meaning the description of the command should be "Starts ...", while each exemple would be in the form of "Start ..."